archetypes/course-section.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+date: {{ .Date }}
```

Comment:
What's date for?

Reply:
The date is part of the hugo boilerplate archetype, not really necessary and can be removed.